install.yml

```
@@ -336,8 +336,37 @@ onAfterClone:
       res = jelastic.dev.scripting.eval("appstore", session, "InstallApp", data);
       return {"result": 0, "out": res}
 
+onBeforeDelete:
+  deleteEnvLink
+
 success: /text/success.md
 
+# -- Actions --
+actions:
+  deleteEnvLink:
+    - script: |-
+        const envName = "${env.envName}";
+        jCustomerEnv = jelastic.env.control.GetNodeGroups(envName, session).object.filter(function (object) {
+                                return object.name == "cp";
+                              }).pop().envLink;
+        if (jCustomerEnv != null) {
+          envsLinked = jelastic.env.control.GetNodeGroups(jCustomerEnv, session).object.filter(function (object) {
+                                return object.name == "cp";
+                              }).pop().envLink;
+          if (envsLinked.indexOf(",") != -1) {
+            envsLinkedArr = envsLinked.split(",");
+            removedEnv = envsLinkedArr.splice(envsLinkedArr.indexOf[envName], 1);
```

This doesn't work at all because you used square brackets instead of parentheses. Did you really test it ? It took me 30 seconds with JSFiddle to see that nothing was working as expected.
```suggestion
            removedEnv = envsLinkedArr.splice(envsLinkedArr.indexOf(envName), 1);
```

---

Also, we need to check `envsLinkedArr` really contains `envName` (i.e. index isn't `-1`), even though it shouldn't happen. But if for whatever reason we have an issue and the Jahia environment was not present in the jCustomer environment `envLink`, this script will randomly remove the last item of the `envLink` list, causing even more trouble (that we will probably have a hard time figuring out).